PFR/ForMathlib/Uniform.lean

```diff
@@ -14,16 +15,29 @@ variable {Ω : Type uΩ} {S : Type uS} {T : Type uT} [mΩ : MeasurableSpace Ω]
 /-- The assertion that the law of $X$ is the uniform probability measure on a finite set $H$.
 While in applications $H$ will be non-empty finite set, $X$ measurable, and and $μ$ a probability
 measure, it could be technically convenient to have a definition that works even without these
-hypotheses. (For instance, `isUniform` would be well-defined, but false, for infinite `H`) -/
+hypotheses. (For instance, `isUniform` would be well-defined, but false, for infinite `H`).
+
+This should probably be refactored, requiring instead that `μ.map X = uniformOn H` -- but at the
+time of writing `uniformOn` did not exist in mathlib. -/
```

Comment:
It did exist though 😅 (under the name `condCount`)

Reply:
If something has a wrong name, it's just like it doesn't exist. Naming things right summons them into existence :-)